app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/PlaceRegisterActivity.kt

```diff
@@ -90,16 +94,23 @@ class PlaceRegisterActivity : AppCompatActivity(), PlaceContract.View,
         toast("$tag")
         imageContainer.removeAllViews()
         uris.forEach { uri ->
-            val iv = LayoutInflater.from(this).inflate(
+            val rl = LayoutInflater.from(this).inflate(
```

Comment:
```suggestion
            val rlSlideImg = LayoutInflater.from(this).inflate(
```
이런식으로 변수에 의미를 담아주면 좋을 것 같아요.

Reply:
수정했습니다